src/openforms/js/components/form/edit/options.js

```diff
@@ -54,6 +54,14 @@ const SHOW_IN_PDF = {
   defaultValue: true,
 };
 
+const AUTOCOMPLETE = {
+  type: 'checkbox',
```

Comment:
https://formio.github.io/formio.js/app/builder -> this is exposed as a textfield in the vanilla formio builder, which allows specifying on/off as textfield.

So rather than using a checkbox (which results in `true`/`false`), it'd probably be better to use a radio or select dropdown with the values `on`/`off`